src/config/logger.ts

```
@@ -58,3 +59,54 @@ if (config.nodeEnv !== "production") {
+const REQUIRED_FIELDS: (keyof FinancialLogPayload)[] = [
+  "event", "amount", "currency", "userId", "accountId",
+  "idempotencyKey", "transactionId", "status", "correlationId",
+];
+
+const CARD_NUMBER_PATTERN = /\b\d{13,19}\b/g;
+
+function redactPii(value: string): string {
+  return value.replace(CARD_NUMBER_PATTERN, "[REDACTED]");
+}
+
+export function logFinancialEvent(payload: Omit<FinancialLogPayload, "timestamp" | "environment"> & Partial<Pick<FinancialLogPayload, "timestamp" | "environment">>): void {
```

ESLint

[error] Replace `payload:·Omit<FinancialLogPayload,·"timestamp"·|·"environment">·&·Partial<Pick<FinancialLogPayload,·"timestamp"·|·"environment">>` with `⏎··payload:·Omit<FinancialLogPayload,·"timestamp"·|·"environment">·&⏎····Partial<Pick<FinancialLogPayload,·"timestamp"·|·"environment">>,⏎`

(prettier/prettier)